taskrunner/host_gateway.go

```diff
@@ -168,6 +163,12 @@ func (agbh *AgentHostGateway) prepRemoteHost(hostObj host.Host, sshOptions []str
 	// run the setup script with the agent
 	if logs, err := hostutil.RunSSHCommand("setup", hostutil.SetupCommand(&hostObj), sshOptions, hostObj); err != nil {
 		event.LogProvisionFailed(hostObj.Id, logs)
+		grip.Errorf("error running setup script on %s", hostObj.Id)
```

Comment:
I'd put the runner name and make this a structured message.

Reply:
done